presto-tests/src/test/java/com/facebook/presto/resourcemanager/TestDistributedClusterStatsResource.java

```diff
@@ -130,17 +133,21 @@ public void testClusterStatsRedirectToResourceManager()
         assertEquals(clusterStats.getRunningTasks(), 12);
     }
 
-    private void waitForGlobalQueryViewInCoordinator(int numberOfRunningQueries, QueryState state, TestingPrestoServer coordinator, long timeoutInMillis)
+    private void waitForGlobalQueryViewInCoordinator(int numberOfRunningQueries, QueryState state, long timeoutInMillis)
```

Comment:
rename `numberOfRunningQueries` to some generic name as this can be running queries or queued queries depends on the state.